src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithDefaultNullFW.java

```diff
@@ -172,6 +172,10 @@ public ListWithDefaultNullFW tryWrap(
         }
         int fieldLimit = offset + FIRST_FIELD_OFFSET;
         bitmask = 0;
```

Comment:
Suggest moving bitmask reset after call the `length()` since fields logically follow after length.